habitat_baselines/rl/ppo/ppo.py

```diff
@@ -152,6 +152,13 @@ def update(self, rollouts: RolloutStorage) -> Tuple[float, float, float]:
 
         return value_loss_epoch, action_loss_epoch, dist_entropy_epoch
 
+    def evaluate_actions(
```

Comment:
Doc string will be useful.

Reply:
Add a little doc string and made it internal since it just exists to be overridden with inheritance.